ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1026,7 +1027,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
                                 required={!this.isMode(WorkspaceEditMode.Duplicate)}>
           <FlexRow style={{alignItems: 'baseline'}}>
             <TextInput type='text' style={styles.textInput} autoFocus placeholder='Workspace Name'
-              value = {name}
+              defaultValue = {name}
```

Comment:
not 100% sure why this change works, but it seems to.  Please LMK if I have misunderstood something here.